templates/isv.html

```diff
@@ -0,0 +1,118 @@
+        <li class="p-list__item is-ticked">Cloud, Kubernetes, edge, infrastructure</li>
+      </ul>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/6c0471ea-Kubernetes-Easier+deployments.svg" alt="" width="400" height="229">
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/af9d1822-Kubernetes-dedicated+platforms.svg" alt="" width="400" height="229">
+    </div>
+    <div class="col-6">
+      <h3>Dedicated platforms improve operations</h3>
+      <p>MicroK8s is the easiest way to consume Kubernetes as it abstracts away much of the complexity of managing the lifecycle of clusters. Its low-touch UX automates or simplifies operations such as deployment, clustering, and enabling of auxiliary services required for a production-grade K8s environment.</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Full control over resources and workloads</li>
+        <li class="p-list__item is-ticked">No noisy neighbours</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Build on MicroK8s, the lightweight Kubernetes</h2>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p>MicroK8s is a zero-ops Kubernetes for delivering your software. Suitable for clouds, edge and IoT clusters, #MicroK8s offers a self-healing high availability feature for enterprise-grade deployments.</p>
+      <h3></h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Fire-and-forget K8s with transactional over-the-air updates</li>
+        <li class="p-list__item is-ticked"><a href="https://microk8s.io/high-availability">Self-healing high availability</a></li>
+        <li class="p-list__item is-ticked">GPU acceleration and other add-ons</li>
+      </ul>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/d942fae6-Kubernetes-over-air-updates.svg" alt="" width="500" height="131">
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/f89832c6-Kubernetes-CNCF+compliant.svg" alt="" width="400" height="229">
+    </div>
+    <div class="col-6">
+      <h3>As you want it</h3>
+      <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking &amp; load balancing, storage, service mesh, observability, GPU &amp; FPGA acceleration, multi-cluster management and more - something unique among lightweight Kubernetes distributions.</p>
```

Comment:
maybe here is `ndash;` 😉
```suggestion
      <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking &amp; load balancing, storage, service mesh, observability, GPU &amp; FPGA acceleration, multi-cluster management and more &ndash; something unique among lightweight Kubernetes distributions.</p>
```